Library/Formula/poppler.rb

```diff
@@ -52,7 +52,9 @@ def install
       --enable-introspection=yes
     ]
 
-    if build.with? "qt"
+    if build.with?("qt") && build.with?("qt5")
+      odie "poppler: --with-qt and --with-qt5 cannot be used at the same time"
```

Comment:
Yeh, I read that. Whether it's rescuable or not is an implementation detail; we're not rescuing it in this case.